### PR TITLE
Refs #34511 - align pf4 design to breadcrumbs

### PR DESCRIPTION
**Pull request by amirfefer**
Following https://github.com/theforeman/foreman/pull/9119#issuecomment-1050231133, this PR fixes the breadcrumbs styling and aligns it to PF4 design.

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.scss [hunk 1]
```diff
@@ -21,3 +21,22 @@
     }
   }
 }
+
+.breadcrumb-bar-pf4 {
+  margin: 0 0 24px;
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.scss]: Should be 16
>  `There should be 16px of padding both between the breadcrumb and the masthead, and between the breadcrumbs and anything that comes below.`

 https://www.patternfly.org/v4/components/breadcrumb/design-guidelines

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.scss [hunk 2]
```diff
@@ -21,3 +21,22 @@
...
+    .breadcrumb-switcher {
+      position: relative;
+      display: inline-block;
+      vertical-align: middle;
+      margin-left: 10px;
+
+      .spinner {
+        float: unset;
+      }
+    }
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.scss]: Unused class, should probably be removed from the other css as well

**Review by MariaAga — CHANGES_REQUESTED** [review]
Thanks! works well just needs a small adjustment

**Comment by amirfefer** [comment]
@MariaAga Thanks, I fixed the css issue, and also added a test

**Comment by amirfefer** [comment]
rebased

**Review by MariaAga — APPROVED** [review]
LGTM

**Comment by MariaAga** [comment]
merged, thanks @amirfefer!

**Pull request merged**